plugin/src/main/java/org/opensearch/ml/plugin/MachineLearningPlugin.java

```diff
@@ -136,7 +136,13 @@
 import com.google.common.collect.ImmutableList;
 
 public class MachineLearningPlugin extends Plugin implements ActionPlugin {
-    public static final String TASK_THREAD_POOL = "OPENSEARCH_ML_TASK_THREAD_POOL";
+    public static final String ML_THREAD_POOL_PREFIX = "thread_pool.ml_commons.";
+    public static final String GENERAL_THREAD_POOL = "opensearch_ml_general";
```

Comment:
This general thread pool is meant for all other tasks that do not have a dedicated thread pool? I see this one is only used for SyncUp Cron jobs for now.

Reply:
Yes, this is for all other tasks like syncup cron.